UnityProject/Assets/Scripts/Items/Others/Magical/ScrollOfTeleportation.cs

```diff
@@ -39,7 +40,14 @@ public void TeleportTo(TeleportDestination destination)
 
 			Transform spawnTransform = PlayerSpawn.GetSpawnForJob((JobType)destination);
 			teleport.ServerTeleportWizard(teleportingPlayer, spawnTransform.position.CutToInt());
-			Chat.AddChatMsgToChat(teleportingPlayer.Player(), $"SCYAR NILA {destination.ToString().ToUpper()}!", ChatChannel.Local);
+
+			SpellData teleportSpell = SpellList.Instance.Spells.Find(spell => spell.Name == "Teleport");
+
+			SoundManager.PlayNetworkedAtPos(
+					teleportSpell.CastSound, teleportingPlayer.Player().Script.WorldPos, sourceObj: teleportingPlayer);
+
+			var incantation = $"{teleportSpell.InvocationMessage.Trim('!')} {destination.ToString().ToUpper()}!";
```

Comment:
Codacy found an issue: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6318997)